docker/build_scripts/requirements3.10.txt

```diff
@@ -1,23 +1,26 @@
-#
-# This file is autogenerated by pip-compile with Python 3.10
-# by the following command:
-#
-#    nox -s update_python_dependencies-3.10
-#
+# This file was autogenerated by uv via the following command:
+#    nox -s update_python_dependencies
 build==1.1.1 \
     --hash=sha256:8ed0851ee76e6e38adce47e4bee3b51c771d86c64cf578d0c2245567ee200e73 \
     --hash=sha256:8eea65bb45b1aac2e734ba2cc8dad3a6d97d97901a395bd0ed3e7b46953d2a31
-    # via -r requirements.in
+importlib-metadata==7.1.0 \
```

Comment:
This seems strange. Why does pip-compile and uv compile behavior differ ?
importlib-metadata was not required before and as far as I can see, build version did not change.

Reply:
https://github.com/pypa/build/blob/b36126a2b7a387757519029d121da17a38135d67/pyproject.toml#L41


Looks like uv resolves for 3.10.0, while pip-compile uses whatever it was running on.